packages/mobile/src/screens/sign-on-screen/components/AccountHeader.tsx

```diff
@@ -151,6 +151,17 @@ const CoverPhoto = (props: CoverPhotoProps) => {
   )
 }
 
+export const ReadOnlyCoverPhotoBanner = () => {
```

Comment:
nice, i wonder if you can just use the CoverPhoto component moving forward cause your user should have an id? this is totes fine for now though considering we arent actually making the user yet